guide/12-enrich-data-with-thematic-information/part1_introduction_to_geoenrichment.ipynb

```diff
@@ -11,139 +11,576 @@
     "# Part 1 - Introduction to GeoEnrichment"
```

Comment:
<h2>Evaluate Results </h2><p>An extremely effective starting point for analysis is simply visualizing the results. Here, we are using <code>matplotlib</code> to visualize the differences between the locations based on the enriched data.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1405/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>